### PR TITLE
added build history component

### PR DESCRIPTION
**Pull request by pawelpaszki**
## Jira

https://issues.jboss.org/browse/AEROGEAR-7712

### The component: 

webp net-resizeimage 1

webp net-resizeimage

### The component around the page

webp net-resizeimage 2

### CODE REVIEW

File: ui/src/components/overview/MobileClientBuildHistoryList.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                        const durationString = this.getDurationString(durationSeconds)
+                        return (
+                            <div className={i === 0 ? 'build-history-item-top': 'build-history-item'}>
+                                <ListGroupItem  key={mobileClientBuild.metadata.uid}>
+                                    <Col md={12}>
+                                        <div >{/* + (i === mobileClientBuilds.length - 1 ? "" : "-top") */}
+                                            <Col md={2}>
+                                                <BuildStatus build={mobileClientBuild}></BuildStatus> <a class-name="incSizeLink">Build #{buildNumber} </a>
+                                            </Col>
+                                            <Col md={2}>
```
Comment (ciaranRoche) [inline ui/src/components/overview/MobileClientBuildHistoryList.js]: I don't know if this is possible with pattern fly but in this `Col` can we have a `ListGroupItem` and split that Item into two columns, I think it should sort out the styling and align the `View Logs`

File: ui/src/components/overview/MobileClientBuildHistoryList.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.build-history-item, .build-history-item-top {
+  padding-top: 1em;
+  padding-bottom: 1em;
+}
+
+.build-history-item-top {
+  border: 1px solid grey;
```
Comment (ciaranRoche) [inline ui/src/components/overview/MobileClientBuildHistoryList.scss]: This grey appears to be a lot darker then the grey we have elsewhere, can we align these.

File: ui/src/components/overview/MobileClientBuildHistoryList.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    render = () => {
+        const mobileClientBuilds = Object.values(this.props.mobileClientBuilds || {});
+        return (
+            <ListGroup>
+                {mobileClientBuilds.map(
+                    ( mobileClientBuild, i ) => {
+                        const phase = mobileClientBuild.status.phase;
+                        const buildNumber = mobileClientBuild.metadata.annotations['openshift.io/build.number'];
+                        const durationSeconds = Math.abs(Math.round(new Date(mobileClientBuild.status.completionTimestamp).getTime()/1000)
```
[the remark this answers is not in the document]
Comment (pawelpaszki) [reply ui/src/components/overview/MobileClientBuildHistoryList.js]: @sedroche I am not sure whether moment can handle it.

**Comment by jhellar** [comment]
I've opened new https://github.com/aerogear/mobile-client-service/pull/33 with Pawel's changes. It is rebased and fixes styling. Closing this PR.

**Pull request closed without merging**